pages/common/rtv.md

```diff
@@ -0,0 +1,21 @@
+# cd
+
+> Reddit Terminal Viewer.
+> Use arrow keys to navigate. Right and Left to view and return from a submission, respectively.
+
+- Open a subreddit:
+
+`/front`
+`/r/tldr`
```

Comment:
The convention is one command per entry. This could be split into "open the reddit frontpage" and "open a subreddit" examples.